### DISCUSSION TITLE
HandlerNotFound Exception when trying to use Maps on Android

### DISCUSSION THREAD

**Discussion by AlbrechtWM** [dotnet/maui#11945] [Q&A]
I keep getting an error about a Handler not being found for view 'Microsoft.Maui.Controls.Maps.Map'.  I understand the map control is not ready for windows devices yet, but this happens on both the Android emulator and on a physical Android phone.  

I got the Nuget package, added Microsoft.Maui.Controls.Maps.Map to my xmlns, everything looks fine.  I am only trying to do <maps:Map/>, simply adding a map control into the main page, and I get the HandlerNotFound exception at runtime everytime.

Any ideas?  Am I missing something obvious?

**Comment by mhrastegari** [comment]
You need to initialize it in `MauiProgram.cs` file.

https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/map?view=net-maui-7.0#map-initialization

**Reply by AlbrechtWM** [reply]
Thank you for the reply.  It was indeed already initialized via the builder.  However this page gives me some more stuff to try.
